dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequest.java

```diff
@@ -2053,19 +2049,17 @@ public String getQueryString()
         if (WCCustomProperties.CHECK_REQUEST_OBJECT_IN_USE){
             checkRequestObjectInUse();
         }
-        SRTServletRequestThreadData reqData = SRTServletRequestThreadData.getInstance();
-        if (reqData.getQueryString()==null && !reqData.isQSSetExplicit())
-            if (_request != null && reqData != null) {
-                reqData.setQueryString(_request.getQueryString());             
-            }
         // 321485
-        String queryString = null;
-        if (reqData != null)
-            queryString = reqData.getQueryString();
         if (TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.FINE)) {  //306998.15
-            logger.logp(Level.FINE, CLASS_NAME,"getQueryString", " queryString --> " + PasswordNullifier.nullifyParams(queryString));
+            logger.logp(Level.FINE, CLASS_NAME,"getQueryString", " queryString --> " + PasswordNullifier.nullifyParams(this._queryString));
         }
```

Comment:
You moved the trace statement to before queryString is set.  You should move it back.

Reply:
oops, my bad, I've put it back